autoapi/mappers/javascript.py

```diff
@@ -35,7 +35,7 @@ def read_file(self, path, **kwargs):
     def map(self, options=None):
         '''Trigger find of serialized sources and build objects'''
         for path, data in self.paths.items():
-            for item in data:
+            for item in (i for i in data if u'files' not in i):
```

Comment:
Please could you add a comment here for why the check is included.
Also, I'm being picky but, please could you move the `if` to inside the for loop. Looping over a generator defined on the loop itself is a bit unconventional.